src/keycloak/index.js

```diff
@@ -1,6 +1,8 @@
 import Keycloak from "keycloak-js";
 
-let keycloak = Keycloak();
+// Keycloak uses "public\keycloak.json" by default if not otherwise specified.
+// https://www.keycloak.org/docs/latest/securing_apps/index.html#_javascript_adapter
+let keycloak = new Keycloak();

```

Comment:
I added the `new` keyword here. I'm not sure why it worked before, but this is how it is created throughout the documentation.

https://www.keycloak.org/docs/latest/securing_apps/index.html#constructor